llvm/test/Transforms/Inline/AArch64/sme-pstateza-attrs.ll

```diff
@@ -85,7 +88,7 @@ define void @new_za_caller_nonza_callee_dont_inline() "aarch64_pstate_za_new" {
 ; CHECK-LABEL: define void @new_za_caller_nonza_callee_dont_inline
 ; CHECK-SAME: () #[[ATTR2]] {
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    call void @nonza_callee()
+; CHECK-NEXT:    call void @inlined_body()
```

Comment:
LINE 87: Should we rename `@new_za_caller_nonza_callee_dont_inline` to `@new_za_caller_nonza_callee_inline` in a similar way to the changes in the other test file? Same goes for other tests below.

Reply:
Done.